diff --git a/node_modules/@theguild/remark-mermaid/dist/mermaid.js b/node_modules/@theguild/remark-mermaid/dist/mermaid.js
index 317b217..ea9a48b 100644
--- a/node_modules/@theguild/remark-mermaid/dist/mermaid.js
+++ b/node_modules/@theguild/remark-mermaid/dist/mermaid.js
@@ -19,7 +19,10 @@ function Mermaid({ chart }) {
         securityLevel: "loose",
         fontFamily: "inherit",
         themeCSS: "margin: 1.5rem auto 0;",
-        theme: isDarkTheme ? "dark" : "default"
+        theme: isDarkTheme ? "dark" : "default",
+        themeVariables: {
+          primaryTextColor: "#1a1a1a"
+        }
       };
       try {
         mermaid.initialize(mermaidConfig);
